src/pages/contributors/yaroslav-afenkin.adoc

```diff
@@ -0,0 +1,96 @@
+= Yaroslav Afenkin
+:page-name: Yaroslav Afenkin
+:page-linkedin: yaroslavafenkin
+:page-twitter:
+:page-github: yaroslavafenkin
+:page-email:
+:page-image: avatar/yaroslav-afenkin.jpg
+:page-pronouns: He/Him/His
+:page-location: Odesa, Ukraine
+:page-firstcommit: 2018
+:page-datepublished: 2025-03-25
+:page-featured: true
+:page-intro: Yaroslav Afenkin is a software engineer based out of Odesa, Ukraine and has been involved with the Jenkins project in multiple instances throughout his career. Whether it was during his first job as a junior engineer, his time working on the Jenkins security team, or his return to contribute to the Jenkins Content Security Policy (CSP) project, Yaroslav has always been intrigued with the abilities of Jenkins. While not necessarily a Jenkins user, his work on securing Jenkins and fixing vulnerabilities has helped the entire community. Thankfully, Yaroslav makes sure to balance his work life with his personal life by finding time to relax and enjoy his free time when he's not deep in the code.
+
+== What is your background prior to contributing to Jenkins?
+
+I went to school in Ukraine and it was heavy on English, so I had about five to seven hours of English a day.
+By the time I finished school, I got to a pretty decent level, which helps a lot in the industry that I'm working in.
+I then entered university in Ukraine, but after studying here for two years, it wasn't as good as I had hoped.
+I decided to move to Warsaw, Poland and continued my studies there.
+With one year left to study, I found a job working as a junior software engineer.
+Initially, I was hired to work on some insurance stuff.
+They were using a suite was called Guidewire.
+However, I was lucky enough that there was no Guidewire project for me.
+
+As a result of this, I landed on a project with a very heavy Java stack, Java Spring, and Angular in the front end.
+That really helped a lot and I feel like I won a lottery there because that's how I developed my most of my Java and Javascript skills.
```

Comment:
```suggestion
That really helped a lot and I feel like I won a lottery there because that's how I developed most of my Java and Javascript skills.
```